celavi/costmethods.py

```diff
@@ -28,7 +37,7 @@ def zero_method(path_dict):
 
         Parameters
         ----------
-        path_dict
+        path_dict : dict
```

Comment:
I assume this is a `Dict[str, float]`, but I may wrong. I suggest documenting the key and value data types as shown in the last sentence.

Reply:
This dictionary keeps growing in size and complexity, and it's also part of the non-data-driven portion of the framework so the structure is allowed to change with the supply chain. What I'll do - and let me know if you think this is a bad idea - is put the dictionary structure and documentation in the .rst file for Cost Methods, and specify that it's for the blade case study. That way the method docstrings are kept short and the dictionary structure doesn't have to be repeated over and over.

Reply:
Ok, I see! As long as the dictionary is documented somewhere, that's great!